magicctapipe/scripts/lst1_magic/config.yaml

```diff
@@ -82,7 +67,7 @@ MAGIC:
         tailcut: [12, 8]
         ring_completeness_threshold: 25
 
-save_images: false
+save_images: true
```

Comment:
while this is required for the cloud correction code to work, saving images increases the size of the files considerable, therefore in the default config file the option should be kept as false